### ISSUE TITLE
Usability/error messages

### ISSUE THREAD

**Issue by olaf-wagner** [initOS/odoo-analyse#5]
Running the program on a repository for the first time results in this output:
```
odoo@36e1c0562d90:/$ odoo_analyse -p /opt/odoo/lib/addons/rental-vertical
Language analyse needs cloc
Language analyse needs cloc
Language analyse needs cloc
...
```

This is fixed under Linux with
```
sudo apt-get install cloc
```

A hint that this is required might be helpful.

Having found that one of the --show options is required, I was then presented with the following error:
```
odoo@36e1c0562d90:~$ odoo_analyse -p /opt/odoo/lib/addons/rental-vertical --show-dependency
odoo@36e1c0562d90:~$ Error: no "view" mailcap rules found for type "application/pdf"
/usr/bin/xdg-open: 778: /usr/bin/xdg-open: www-browser: not found
/usr/bin/xdg-open: 778: /usr/bin/xdg-open: links2: not found
/usr/bin/xdg-open: 778: /usr/bin/xdg-open: elinks: not found
/usr/bin/xdg-open: 778: /usr/bin/xdg-open: links: not found
/usr/bin/xdg-open: 778: /usr/bin/xdg-open: lynx: not found
/usr/bin/xdg-open: 778: /usr/bin/xdg-open: w3m: not found
xdg-open: no method available for opening 'module.gv.pdf'
```

I finally succeeded with installing links2. However, a hint that one of those packages is required would be helpful, too.

**Comment by OSevangelist** [comment]
@fkantelberg would you mind to review @olaf-wagner remarks and add a hint to the README ?

**Comment by fkantelberg** [comment]
@olaf-wagner @OSevangelist 
The dependency for cloc is missing I agree but the other dependencies are more vague because you just need a viewer and this is platform dependent. The only thing I can do here which would be generic is save the output to a file and not show it directly.

**Comment by olaf-wagner** [comment]
On Wed, 02 Dec 2020 07:58:47 -0800
fkantelberg <notifications@github.com> wrote:

> @olaf-wagner @OSevangelist 
> The dependency for cloc is missing I agree but the other dependencies are more vague because you just need a viewer and this is platform dependent. The only thing I can do here which would be generic is save the output to a file and not show it directly.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/initOS/odoo-analyse/issues/5#issuecomment-737322730

Perhaps adding a -o option for specifying an output file would be an idea
then. Or just mentioning the pre-requisites in the documentation.
I'm not insisting on any of it of course.

Olaf
-- 
Olaf Wagner -- elego Software Solutions GmbH -- http://www.elegosoft.com 
               Gustav-Meyer-Allee 25 / Gebäude 12, 13355 Berlin, Germany
phone: +49 30 23 45 86 96  mobile: +49 177 2345 869  fax: +49 30 23 45 86 95
Geschäftsführer: Olaf Wagner | Sitz: Berlin
Handelregister: Amtsgericht Charlottenburg HRB 77719 | USt-IdNr: DE163214194

**Comment by fkantelberg** [comment]
I added the dependencies to the readme and the tool will now render to a file instead. The changes are released in the version 0.9.4.
